src/app/services/validation.service.ts

```diff
@@ -0,0 +1,28 @@
+export class ValidationService {
+  static getValidatorErrorMessage(validatorName: string, validatorValue?: any) {
+    let config = {
+      'required': 'Required',
+      'invalidWord': 'invalid word name , only accept alphabets',
+      'invalidPronounce': 'invalid pronounce ,only accept lowercase alphabet and dashes',
+      'maxlength': `Maximum length ${validatorValue.requiredLength}`
+    };
+    return config[validatorName];
+  }
+
+  static wordValidator(control) {
+    if (control.value.match(/^[a-zA-Z]*$/)) {
```

Comment:
the RegEx should be in an object in the top of the file, not scattered everywhere, put it in an object and use it here and everywhere

Reply:
Fixed